aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js

```diff
@@ -0,0 +1,87 @@
+import React, { useContext } from 'react';
+import { Box, Grid, Header, Heading, ResponsiveContext } from 'grommet';
+import {
+  AppContainer,
+  PageContainer,
+  PageContainerContext,
+} from '../page-layouts/components';
+import { ContentArea } from '../page-layouts/anatomy/components';
+
+const parentGridColumns = {
+  xsmall: '100%',
+  small: '100%',
+  medium: ['auto', 'auto'],
+  large: ['auto', 'medium'],
+  xlarge: ['auto', 'medium'],
+};
+
+const firstChildGridColumns = {
+  xsmall: '100%',
+  small: '100%',
+  medium: '100%',
+  large: ['auto', 'small'],
+  xlarge: ['auto', 'small'],
+};
+
+const secondChildGridColumns = {
+  xsmall: '100%',
+  small: '100%',
+  medium: '100%',
+  large: { count: 'fit', size: ['small', 'auto'] },
+  xlarge: { count: 'fit', size: ['small', 'auto'] },
+};
+
+export const ResponsiveContentLayoutExample = () => (
+  <AppContainer background="background-back">
+    <ContentArea title="Global Header" />
+    <PageContainer>
+      <PageHeader />
+      <PageContent />
+    </PageContainer>
+  </AppContainer>
```

Comment:
Reenforcement, plus will include a gap below the PageContainer

```suggestion
    </PageContainer>
    <ContentArea title="Global Footer" />
  </AppContainer>
```

Reply:
Running into an issue with our Example container here that seems to not let the child (in our case AppContainer) grow beyond the height of the example. Need to investigate a little further tomorrow.